.ci/azure/linux.yml

```diff
@@ -372,7 +372,6 @@ jobs:
   # TODO Reenable PDPD after paddlepaddle==2.5.1 with compliant protobuf is released (ticket 95904)
   - script: $(RUN_PREFIX) $(INSTALL_TEST_DIR)/paddle_tests --gtest_print_time=1 --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-Paddle.xml
     displayName: 'Paddle Frontend UT'
-    enabled: 'false'
```

Comment:
please, uncomment this line https://github.com/openvinotoolkit/openvino/blob/120d649d56a95648d8470ab4d708717aa2d68773/.ci/azure/linux.yml#L169 to install tests dependencies, used to generate PDPD models from code